test.sh

```diff
@@ -1,10 +1,25 @@
+#!/bin/sh
+
+version="0.0.1"
+what_arch=`uname -m`
+
+if [ ${what_arch} == "x86_64" ]
+then
+    arch="amd64"
+elif [ ${what_arch} == "arm64"]
+then
+    arch=${what_arch}
+fi
```

Comment:
Different OS's use different identifiers here.

| OS | 64-bit Intel | 64-bit ARM |
|-|-|-|
| Red Hat family | `x86_64` | `aarch64` |
| Debian family | `amd64` | `arm64` |
| macOS | `x86_64` | `arm64` |
| Windows | ??? | N/A |

You probably need to compare (pseudocode):

```bash
if x86_64 || amd64; then amd64
if aarch64 || arm64; then arm64
```

Reply:
Will amend, thanks